dev.rollup.config.js

```diff
@@ -37,6 +35,18 @@ const buildExample = (relativeFilePath) => {
     );
 };
 
+const tsc = () => {
+    return new Promise((resolve) => {
+        const bin = require.resolve('typescript/bin/tsc');
+        spawn(bin, [
+            '--incremental',
+            '--outDir', tmpDestPath,
```

Comment:
Should we output this to a `bin` or `tsbin ` to avoid any overrides?  Not sure if it's really necessary though.

Reply:
My initial thought here was that as part of the dev server the compiled declaration files would be output along with the bundles (similar to our actual build process). It's a good idea to output them to a subfolder though. The only reason that we have to output them at all is to allow for incremental builds, which is just an optimization